File: app/src/main/java/com/ndipatri/solarmonitor/services/solar/dto/LastMonthData.java

package com.ndipatri.solarmonitor.services.solar.dto;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LastMonthData {

    @SerializedName("energy")
    @Expose
    private Double energy;

    public Double getEnergy() {
        return energy;
    }

    public void setEnergy(Double energy) {
        this.energy = energy;
    }

}

File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/LastMonthData.kt
package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LastMonthData {

    @SerializedName("energy")
    @Expose
    var energy: Double? = null

}
